src/Debugger/Breakpoint.php

```diff
@@ -548,30 +581,111 @@ public function validate()
             );
             return false;
         }
+        return true;
+    }
+
+    private function validateSourceLocation()
```

Comment:
Can we add a comment that explain the line validation is not perfect with some false negative cases?